src/components/world/World.js

```diff
@@ -95,11 +95,11 @@ class World extends Component {
     if (scroll && loaded && !prevProps.loaded) {
       scroll.scrollTo(scrollX, scrollY);
     }
-
-    if (
-      onlyMatchingScene &&
-      onlyMatchingScene !== prevProps.onlyMatchingScene
-    ) {
+    
+    const sceneMatchesPrevious = !!(onlyMatchingScene !== null &&
+       prevProps.onlyMatchingScene !== null && 
+       onlyMatchingScene.id === prevProps.onlyMatchingScene.id);
```

Comment:
Close :-) but I think this should have been `onlyMatchingScene.id !== prevProps.onlyMatchingScene.id` as this scrollTo section should only fire the first time this specific scene becomes the only matching scene

Reply:
I'm also getting some weird functionality with this PR where after selecting a scene I can no longer scroll the world view, it just instantly keeps snapping back to the current position